src/components/VaccineDetails/VaccineDetails.tsx

```diff
@@ -0,0 +1,61 @@
+import { StateVaccineData, useStateVaccineData } from 'cms-content/vaccines';
+import { getStateName, Region } from 'common/regions';
+import { Subtitle1 } from 'components/Typography';
+import { HeaderCopy, BorderedContainer } from './VaccineDetails.style';
+import React from 'react';
+import ExternalLink from 'components/ExternalLink';
+import { Grid } from '@material-ui/core';
+import { assert } from 'common/utils';
+
+interface VaccineDetailsProps {
+  data?: StateVaccineData;
+  locationName: string;
+}
+
+export const VaccineDetailsContent = ({
+  data,
+  locationName,
+}: VaccineDetailsProps) => {
+  return (
+    <div>
+      <div>
+        <HeaderCopy>How can I get vaccinated</HeaderCopy>
+        <Subtitle1>in {locationName}</Subtitle1>
+      </div>
+      {data && (
+        <BorderedContainer>
+          <Grid item xs={12}>
+            <p>Here are the best resources we've found to:</p>
+          </Grid>
+          <Grid item xs={12}>
+            <ExternalLink href={data.generalInformationUrl}>
+              Find out about your eligibility
+            </ExternalLink>
+          </Grid>
+          <Grid item xs={12}>
+            <ExternalLink href={data.howToGetVaccinatedUrl}>
+              Sign up for the COVID-19 vaccine
+            </ExternalLink>
+          </Grid>
+        </BorderedContainer>
+      )}
```

Comment:
we probably don't want to show the component at all if we don't have data, unless we have fallback links (from the CDC maybe?) for states that don't have vaccination data